homeassistant/components/bmw_connected_drive/notify.py

```diff
@@ -84,6 +81,4 @@ def send_message(self, message: str = "", **kwargs: Any) -> None:
 
                 vehicle.remote_services.trigger_send_poi(location_dict)
             else:
```

Comment:
A future improvement: Invert the if check above and raise the ValueError if True. Then we can outdent below.

Reply:
Yes, that makes sense. Will do that when your larger change to services/notify is merged so we have to update it only once.